app/views/content/funding-and-support/scholarships-and-bursaries.md

```diff
@@ -47,23 +52,41 @@ Postgraduate teaching bursaries and scholarships are only available for the subj
 | **Maths**                                              | £27,000  | £29,000          |
 | **Physics**                                            | £27,000  | £29,000          |
 
- To be eligible for a bursary, you’ll need at least one of the following:
+### Bursaries
+
+You do not need to apply for a bursary. If you’re eligible, you’ll automatically receive it.
+
+To be eligible for a bursary, you’ll need at least one of the following:
+
+* a bachelor’s degree class 2:2 (honours) or higher
+* a master’s degree
+* a PhD
 
-- a bachelor’s degree class 2:2 (honours) or higher
-- a master’s degree
-- a PhD
+If your course combines 2 subjects (for example, English and history), your course content must be made up of at least 50% of the bursary subject to be eligible.
```

Comment:
Can this be tweaked slightly to make it a little more active?

"If your course combines 2 subjects (for example, English and history), 50% of your course content must be made up of the bursary subject to be eligible."

Reply:
I'm not sure this is easier to understand but will leave for @EllieNodder !

NB we need to get 'at least' in there whichever way round it is written